### PR TITLE
Improved navbar

### LINKED ISSUE #274: [FEATURE] Improve navbar of website
### Description

I want to improve the UI of  navbar of the website. Please assign it to me @The-Shivam-garg .

### Screenshots

ss6

### PR DESCRIPTION
**Pull request by Areebah03**
*Fixed #274 

## 🛠️ Fixes Issue

Fixes #274 

## ✔️ Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## 📄 Note to reviewers

I have improved the navbar of the website by changing fonts, colors and hover effects.

## 📷 Screenshots

| Before | After |
| --- | ---- |
| Capture | ss1 |

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -585,15 +587,21 @@ a:hover{
...
+   
+    font-family: serif;
+}
+header .nav li a:hover{
+    transition: all 0.5s ease-in-out;
+   background-color:#192e3a ;
+   padding: 4px 20px;
+   transform: scale(1.5);
+   border-radius: 50px 0px 50px 0px;
+}
```
Comment (sakshamgurbhele) [inline style.css:607]: can you make the sides circular? not everyone will like this leafy / edged design tbh.

**Review by sakshamgurbhele — CHANGES_REQUESTED** [review]

**Comment by Areebah03** [comment]
@sakshamgurbhele I have made the sides circular. Kindly view it.

File: style.css [hunk 2]
```diff
@@ -595,13 +595,13 @@ header .logo h3{
...
     font-family: serif;
 }
 header .nav li a:hover{
-    transition: all 0.5s ease-in-out;
+    transition: all 0.4s ease-in-out;
    background-color:#192e3a ;
-   padding: 4px 20px;
-   transform: scale(1.5);
-   border-radius: 50px 0px 50px 0px;
+   border-radius: 50px 50px 50px 50px;
```
Comment (sakshamgurbhele) [inline style.css:606]: 👍🏼

**Review by sakshamgurbhele — APPROVED** [review]

**Comment by sakshamgurbhele** [comment]
@Areebah03 Please Resolve the conflicts, so that we can merge it 🚀

**Comment by Areebah03** [comment]
@sakshamgurbhele I have resolve the merge conflicts.

**Pull request merged**